firefox-ios/Client/Configuration/version.xcconfig

```diff
@@ -1 +1 @@
-APP_VERSION = 138.0
+APP_VERSION = 139.0
```

Comment:
Do we want to commit this? I'm also getting this change repeatedly in my local branches

Reply:
Yes we can, I asked here for reference: https://mozilla.slack.com/archives/C03PKCHHSSD/p1744034660940369?thread_ts=1743716249.782939&cid=C03PKCHHSSD